core/src/main/java/feast/core/validators/FeatureSetValidator.java

```diff
@@ -27,13 +27,19 @@
 import java.util.stream.Collectors;
 
 public class FeatureSetValidator {
+
+  private static final String EMPTY_LABEL_KEYS_MESSAGE = "Label keys must not be empty";
```

Comment:
Not sure about the adding a constant `EMPTY_LABEL_KEYS_MESSAGE` to deduplicate the message here.
It actually might be useful to know when if the to error occurs on the featureset or the feature when debugging.

Reply:
I see your point. Changed.